tensorflow/core/platform/default/gpu/cupti_wrapper.h

```diff
@@ -20,10 +20,10 @@ limitations under the License.
 
 #include <stddef.h>
 #include <stdint.h>
-#if defined(WIN32)
-#include "extras/CUPTI/include/cupti.h"
-#else
+#if defined(PLATFORM_GOOGLE)
```

Comment:
I am not very sure that this is good way to set correct platform. In typical cuda usage, the prepending cuda folder is not necessary since the cuda SDK is setted during dependency configuration.